Docs/source/usage/parameters.rst

```diff
@@ -1153,6 +1153,15 @@ Laser initialization
     value of the `By` component is set to zero.
     Note that the current implementation of the parser for B-field on particles
     is applied in cartesian co-ordinates as a function of (x,y,z) even for RZ.
+    To apply a series of plasma lenses, use the option ``repeated_plasma_lens``. This
+    option requires the following parameters,
+    ``repeated_plasma_lens_period``, the period length of the repeat, a single float number,
+    ``repeated_plasma_lens_starts``, the start of each lens relative to the period, an array of floats,
+    ``repeated_plasma_lens_lengths``, the length of each lens, an array of floats,
+    ``repeated_plasma_lens_strengths_B``, the focusing strength of each lens, an array of floats.
+    The applied field is uniform longitudinally (along z) with a hard edge,
+    where residence corrections are used for more accurate field calculation.
+    The field is of the form Bx = strength*y and By = strength*x, Bz = 0.
```

Comment:
Using verbatim to avoid accidental formatting:
```suggestion
    The field is of the form ``Bx = strength*y`` and ``By = strength*x``, ``Bz = 0``.
```

Alternatively, you can also use proper LaTeX math like this :)
```suggestion
    The field is of the form :math:`B_x = \mathrm{strength} \cdot y` and :math:`B_y = \mathrm{strength} \cdot x`, :math`:B_z = 0`.
```